ruby-challenges.rb

```diff
@@ -2,27 +2,58 @@
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
-
 reposts1 = 7
 # Expected output: '7 is odd'
 reposts2 = 42
 # Expected output: '42 is even'
 reposts3 = 221
 # Expected output: '221 is odd'
 
+# Psuedo Code
+# Create a function called is_even_odd
+# Function will take a number as an input
+# Use the .even? method to return a boolean "true/false"
+# Use a conditional to that compares the boolean value returned from the .even? method
+# If number is true then return "#{num} is even"
+# if number is false then return "#{num} is odd"
+
+def is_even_odd num
+    if num.even? == true
+        "#{num} is even"
+    else 
+        "#{num} is odd"
+    end
+end
 
+p is_even_odd reposts1
+# => "7 is odd"
+p is_even_odd reposts2
+# =>"42 is even"
+p is_even_odd reposts3
+# => "221 is odd"
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+# Pseudo Code
+# create a method called gutted_out 
+# Method will take a string as an input example "Rubber Soul"
+# using the .delete! 
+
 beatles_album1 = 'Rubber Soul'
 # Expected output: 'Rbbr Sl'
 beatles_album2 = 'Sgt Pepper'
 # Expected output: 'Sgt Pppr'
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def gutted_out str 
+    str.delete "a,i,e,o,u"
+end
 
```

Comment:
Nice job! Keep in mind handling edge cases, such as the `beatles_album3` variable. It contains a capital vowel, which your method is not accounting for.